src/content/docs/apm/agents/net-agent/net-agent-api/net-agent-api.mdx

```diff
@@ -0,0 +1,2579 @@
+
+    The following methods are available on `ITransaction`:
+
+    <table>
+    <thead>
+        <tr>
+        <th>
+            Name
+        </th>
+
+        <th>
+            Description
+        </th>
+        </tr>
+    </thead>
+
+    <tbody>
+        <tr>
+        <td>
+            [`AcceptDistributedTraceHeaders`](#acceptdistributedtraceheaders)
+        </td>
+
+        <td>
+            Accepts incoming trace context headers from another service.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            [`InsertDistributedTraceHeaders`](#insertdistributedtraceheaders)
+        </td>
+
+        <td>
+            Adds trace context headers to an outgoing request.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            [`AcceptDistributedTracePayload` (obsolete)](#acceptdistributedtracepayload)
+        </td>
+
+        <td>
+            Accepts an incoming distributed trace payload from another service.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            [`CreateDistributedTracePayload` (obsolete)](#createdistributedtracepayload)
+        </td>
+
+        <td>
+            Creates a distributed trace payload for inclusion in an outgoing request.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            [`AddCustomAttribute`](#addcustomattribute)
+        </td>
+
+        <td>
+            Add contextual information from your application to the current transaction in form of attributes.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            [`CurrentSpan`](#currentspan)
+        </td>
+
+        <td>
+            Provides access to the currently executing [span](/docs/agents/net-agent/net-agent-api/ispan), which provides access to span-specific methods in the New Relic API.
+        </td>
+        </tr>
+
+        <tr>
+        <td>
+            [`SetUserId`](#setuserid)
+        </td>
+
+        <td>
+            Associates a User Id to the current transaction.
+        </td>
+        </tr>
+    </tbody>
+    </table>
+    <CollapserGroup>
+    <Collapser
+        id="AcceptDistributedTraceHeaders"
+        title="AcceptDistributedTraceHeaders"
+    >
+        `ITransaction.AcceptDistributedTraceHeaders` is used to instrument the called service for inclusion in a distributed trace. It links the spans in a trace by accepting a payload generated by [`InsertDistributedTraceHeaders`](/docs/agents/nodejs-agent/api-guides/nodejs-agent-api#transaction-handle-insertDistributedTraceHeaders) or generated by some other W3C Trace Context compliant tracer. This method accepts the headers of an incoming request, looks for W3C Trace Context headers, and if not found, falls back to New Relic distributed trace headers. This method replaces the deprecated [`AcceptDistributedTracePayload`](/docs/agents/net-agent/net-agent-api/itransaction#acceptdistributedtracepayload) method, which only handles New Relic distributed trace payloads.
```

Comment:
For other methods, the syntax appears first and then description. Here description precedes syntax, and there's no short summaruy, only the long description